packages/application-shell/src/components/locale-switcher/locale-switcher.tsx

```diff
@@ -97,16 +92,15 @@ const LocaleSwitcher = (props: Props) => {
         }))}
         components={{
           SingleValue: (valueProps: SingleValueProps) => (
-            <SingleValue
-              {...valueProps}
-              localeCount={props.availableLocales.length}
-            />
+            <SingleValue {...valueProps} />
```

Comment:
Nit: I think you can remove the function here, and pass the reference to `SingleValue` directly.

```ts
components={{
  SingleValue,
}}
```

Reply:
Thanks @emmenko! Was this what you had in mind? [44c5bb4](https://github.com/commercetools/merchant-center-application-kit/pull/3293/commits/44c5bb48b4c484deb864b25839c5c72d40ef5010)

Reply:
Yes thanks